optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/roster/RosterRestServiceImpl.java

```diff
@@ -193,8 +190,10 @@ protected EmployeeRosterView getEmployeeRosterView(Integer tenantId, LocalDate s
                 .getResultList();
         Map<Long, List<ShiftView>> employeeIdToShiftViewListMap = new LinkedHashMap<>(employeeList.size());
         for (Shift shift : shiftList) {
-            employeeIdToShiftViewListMap.computeIfAbsent(shift.getEmployee().getId(), k -> new ArrayList<>()).add(
-                    new ShiftView(shift));
+            if (shift.getEmployee() != null) {
```

Comment:
(important to think about) Shouldn't the EmployeeRosterView know the shifts of the employee "unassigned" (so employee = null)? Even if the UI doesn't use it, it's interesting to include in the REST api.
Remember that the end-user might hire external temp-nurses to fill in the unassigned shifts, so he'll want to know how many there are.